app/src/main/java/com/tugraz/quizlet/backend/database/DBManager.kt

```diff
@@ -1,54 +1,94 @@
 package com.tugraz.quizlet.backend.database
 
+import androidx.annotation.UiThread
 import com.google.common.collect.ImmutableList
 import com.tugraz.quizlet.backend.database.model.Question
-import com.tugraz.quizlet.backend.database.model.Question_category
-
 import io.realm.Realm
-import io.realm.RealmList
+import io.realm.RealmConfiguration
+import io.realm.RealmObject
 import io.realm.RealmResults
+import io.realm.kotlin.where
 import io.realm.mongodb.App
 import io.realm.mongodb.AppException
 import io.realm.mongodb.Credentials
 import io.realm.mongodb.mongo.MongoClient
 import io.realm.mongodb.mongo.MongoCollection
 import io.realm.mongodb.mongo.MongoDatabase
 import io.realm.mongodb.sync.SyncConfiguration
+import kotlinx.coroutines.awaitAll
 import org.bson.Document
 import org.bson.types.ObjectId
 import java.util.logging.Logger
 import kotlin.jvm.Throws
-import kotlin.math.absoluteValue
+import com.tugraz.quizlet.backend.database.model.User as User
 
 class DBManager(private val quizletApp: App) : DBInterface {
     companion object {
         val LOG: Logger = Logger.getLogger(DBManager::class.java.name)
     }
 
+    private var anon: io.realm.mongodb.User? = null
+
     private var user: io.realm.mongodb.User? = null
     private var realm: Realm? = null
 
     init {
+        //addUser("emilia0@test.com", "itsmeemilia")
```

Comment:
please remove code which is not needed anymore. If we need it we can always use the git history

Reply:
sure